tools/pubsys-setup/Cargo.toml

```diff
@@ -10,10 +10,10 @@ publish = false
 hex = "0.4.0"
 log = "0.4"
 pubsys-config = { path = "../pubsys-config/" }
-reqwest = { version = "0.10.1", default-features = false, features = ["rustls-tls", "blocking"] }
+reqwest = { version = "0.11", default-features = false, features = ["rustls-tls", "blocking"] }
```

Comment:
Can we hold off on this (and in buildsys) until we do a bigger tokio-dep update?